src/Blink.php

```diff
@@ -56,6 +56,22 @@ public function userCreate(array $user)
         return $this->responseSuccessful($response);
     }
 
+    /**
+     * Send a Post request to the Blink /events/user-password-reset endpoint.
+     *
+     * To notify Blink to create a Customer.io password_reset event.
+     *
+     * @param array $passwordReset - The array containing password_reset event fields.
+     * @return bool
+     */
+    public function userPasswordReset(array $passwordReset)
+    {
+        $response = $this->post('v1/events/user-password-reset', $passwordReset);
+
+        // TODO: throw an exception if the post returns a validation error.
```

Comment:
This should already be handled [down below](https://github.com/DoSomething/gateway/blob/83e0aa2171148a874ae5a41376946f19fce1048a/src/Blink.php#L109-L127)!

Reply:
I copy/pasta'd 🍝 -- can clean this up in a future branch.